Description: Skip test_fetch_package_version
 This test doesn't work in the pybuild environment.
 .
 raven (5.18.0-1) UNRELEASED; urgency=medium
 .
   * Initial release.
Author: Gilles Dubuc <gilles@wikimedia.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

Index: raven-5.18.0/tests/versioning/tests.py
===================================================================
--- raven-5.18.0.orig/tests/versioning/tests.py
+++ raven-5.18.0/tests/versioning/tests.py
@@ -34,6 +34,7 @@ def test_fetch_git_sha():
     ).decode('latin1').strip()
 
 
+@pytest.mark.skipif('True')
 def test_fetch_package_version():
     result = fetch_package_version('raven')
     assert result is not None
